src/repositorios/injury.test.js

```diff
@@ -87,7 +90,8 @@ describe(
                                             id: expect.any(Number),
                                             nome: expect.any(String),
                                             detalhes: expect.any(String),
-                                            grade: expect.any(Number)
+                                            grade: expect.any(Number),
+                                            id_collection: expect.any(Number)
```

Comment:
Test might fail if collection doesn't exist. Fix it using https://github.com/CRICDatabase/searchable-image-database-nodejs/issues/136.